src/server/apollo.ts

```diff
@@ -0,0 +1,93 @@
+import * as Sentry from '@sentry/node';
+import express from 'express';
+import http from 'http';
+import { expressMiddleware } from '@apollo/server/express4';
+import { ApolloServer } from '@apollo/server';
+import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHttpServer';
+import { buildSubgraphSchema } from '@apollo/subgraph';
+import { ApolloServerPluginLandingPageGraphQLPlayground } from '@apollo/server-plugin-landing-page-graphql-playground';
+import { ApolloServerPluginLandingPageDisabled } from '@apollo/server/plugin/disabled';
+import { sentryPlugin, errorHandler } from '@pocket-tools/apollo-utils';
+import config from '../config';
+import { ContextManager } from './context';
+import { readClient } from '../database/client';
+import {
+  initItemEventHandlers,
+  itemsEventEmitter,
+  snowplowEventHandler,
+  sqsEventHandler,
+  unifiedEventHandler,
+} from '../businessEvents';
+import queueDeleteRouter from './routes/queueDelete';
+import { BatchDeleteHandler } from '../aws/batchDeleteHandler';
+import { EventEmitter } from 'events';
+import { initAccountDeletionCompleteEvents } from '../aws/eventTypes';
+import { typeDefs } from './typeDefs';
+import { resolvers } from '../resolvers';
+
+export async function startServer(port: number) {
+  Sentry.init({
+    ...config.sentry,
+    debug: config.sentry.environment == 'development',
+  });
+
+  const app = express();
+  // Our httpServer handles incoming requests to our Express app.
+  // Below, we tell Apollo Server to "drain" this httpServer,
+  // enabling our servers to shut down gracefully.
+  const httpServer = http.createServer(app);
+  // Initialize routes
+  app.use('/queueDelete', queueDeleteRouter);
+
+  // Start BatchDelete queue polling if not test env
+  if (process.env.NODE_ENV != 'test') {
```

Comment:
Nit, grab this from `config.app.environment`, especially if we start passing it as a parameter.

Reply:
sure, although I will probably say we should follow up in a separate ticket since we are using node env elsewhere and I was trying to limit unrelated changes